source/manual/querying-cdn-logs.html.md

```diff
@@ -0,0 +1,183 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Querying CDN Logs
```

Comment:
Perhaps "Query CDN logs" to make it conform to [the styleguide](https://docs.publishing.service.gov.uk/manual/docs-style-guide.html#title).